packages/fluentui/docs/package.json

```diff
@@ -46,6 +46,7 @@
     "react-textarea-autosize": "7.0.0",
     "react-virtualized": "^9.21.1",
```

Comment:
Can be this dependency removed now?

Reply:
Nah.... it's in virtualized table prototype :D